src/hermezjs-typings.d.ts

```diff
@@ -11,6 +11,7 @@
 
 declare module "@hermeznetwork/*" {
   import { BigNumber } from "ethers";
+  import { TxTypesUnion, TxStatesUnion } from "@hermeznetwork/hermezjs/src/enums";
```

Comment:
Why the `Union` suffix? I think that it would make more sense to call them just `TxType` and `TxState`.